src/test/ReactTestUtils.js

```diff
@@ -452,6 +455,12 @@ function makeSimulator(eventType) {
       EventPluginHub.enqueueEvents(event);
       EventPluginHub.processEventQueue(true);
     });
+    // Normally extractEvent enqueues a state restore, but we'll just always do
+    // that.
```

Comment:
Why not leave it in ChangeEventPlugin only for consistency with the non-test env?